about us/about_us.js

```diff
@@ -0,0 +1,341 @@
+      obstacleNameBoxElem.classList.add("obstacle");
+
+      const obstacleNameElem = document.createElement("h3");
+      obstacleNameElem.append(obstacle.obstacleName);
+      obstacleNameBoxElem.append(obstacleNameElem);
+      obstacleNameBoxElem.id = `obstacle${mutualId}`;
+      if (quantity < 4) {
+        firstColumn.append(obstacleNameBoxElem);
+      } else {
+        secondColumn.append(obstacleNameBoxElem);
+      }
+      knownObstaclesBox.append(firstColumn);
+      knownObstaclesBox.append(secondColumn);
+      quantity++;
+    }
+
+    const withSubcategoriesBox = document.createElement("ul");
+    withSubcategoriesBox.classList.add("hide");
+    if (obstacle.hasOwnProperty("obstacleSubCategories")) {
+      const subcategoriesArray = obstacle.obstacleSubCategories;
+      for (let i = 0; i < subcategoriesArray.length; i++) {
+        const listItem = document.createElement("li");
+        withSubcategoriesBox.id = `ul${mutualId}`;
+        if (subcategoriesArray[i].hasOwnProperty("name")) {
+          const subcategoryName = document.createElement("h4");
+          subcategoryName.append(subcategoriesArray[i].name);
+          listItem.append(subcategoryName);
+          withSubcategoriesBox.append(listItem);
+        }
+
+        if (subcategoriesArray[i].hasOwnProperty("subCategoryExplained")) {
+          const subcategoryExplained = document.createElement("p");
+          subcategoryExplained.append(
+            subcategoriesArray[i].subCategoryExplained
+          );
+          listItem.append(subcategoryExplained);
+          withSubcategoriesBox.append(listItem);
+        }
+
+        if (subcategoriesArray[i].hasOwnProperty("obstacleFix")) {
+          const obstacleFix = document.createElement("p");
+          obstacleFix.append(subcategoriesArray[i].obstacleFix);
+          listItem.append(obstacleFix);
+          withSubcategoriesBox.append(listItem);
+
+          knownObstaclesBox.append(withSubcategoriesBox);
+        }
+      }
+    }
+
+    const withoutSubcategoriesBox = document.createElement("ul");
+    withoutSubcategoriesBox.classList.add("hide");
+    if (obstacle.hasOwnProperty("obstacleExplained")) {
+      const obstacleExplained = document.createElement("p");
+      obstacleExplained.append(obstacle.obstacleExplained);
+      const listItem = document.createElement("li");
+      listItem.append(obstacleExplained);
+      withoutSubcategoriesBox.append(listItem);
+      knownObstaclesBox.append(withoutSubcategoriesBox);
+    }
+
+    if (obstacle.hasOwnProperty("obstacleFix")) {
+      const obstacleFix = document.createElement("p");
+      obstacleFix.append(obstacle.obstacleFix);
+      const listItem = document.createElement("li");
+      withoutSubcategoriesBox.id = `ul${mutualId}`;
+
+      listItem.append(obstacleFix);
+      withoutSubcategoriesBox.append(listItem);
+      knownObstaclesBox.append(withoutSubcategoriesBox);
+    }
+
+    mutualId++;
```

Comment:
ids are unique per element. Multiple elems should not share an id. That's what classes are for.